src/Microsoft.AspNetCore.Testing/TestPlatformHelper.cs

```diff
@@ -2,22 +2,22 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.Extensions.PlatformAbstractions;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.AspNetCore.Testing
 {
     public static class TestPlatformHelper
     {
         public static bool IsMono =>
-            PlatformServices.Default.Runtime.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase);
+            Microsoft.Extensions.Internal.RuntimeEnvironment.RuntimeType.Equals("Mono", StringComparison.OrdinalIgnoreCase);
```

Comment:
Let's change this to `Type.GetType("Mono.Runtime") != null`. Would be nicer than comparing strings.